bin/install_cocoapods

```diff
@@ -32,8 +32,20 @@ else
 fi
 
 function lockfile_error () {
-  message="\`Podfile.lock\` was changed by \`bundle exec pod install\`. Please run \`bundle exec pod install\` locally with a clean cache and commit the result."
+  message=$(cat <<EOF
+  \`Podfile.lock\` was changed by \`bundle exec pod install\`. Please run \`bundle exec pod install\` locally with a clean cache and commit the result.
+
+  <details><summary>See diff</summary>
+
+  \`\`\`
```

Comment:
Nitpick: According to the documentation, it seems that `buildkite-agent annotate` [supports GitHub-Flavored Markdown](https://buildkite.com/docs/agent/v3/cli-annotate#creating-an-annotation-description)… and thus should support syntax highlighting?
```suggestion
  \`\`\`diff
```

Reply:
See #61